sdk/keyvault/azure-security-keyvault-administration/src/samples/java/com/azure/security/keyvault/administration/AccessControlHelloWorld.java

```diff
@@ -23,10 +23,10 @@ public class AccessControlHelloWorld {
      */
     public static void main(String[] args) {
         /* Instantiate a KeyVaultAccessControlClient that will be used to call the service. Notice that the client is
-        using default Azure credentials. To make default credentials work, ensure that environment variables
-        'AZURE_CLIENT_ID', 'AZURE_CLIENT_KEY' and 'AZURE_TENANT_ID' are set with the service principal credentials.
+        using default Azure credentials. To make default credentials work, ensure that the environment variable
+        'AZURE_CLIENT_ID' is set with the principal ID of a managed identity that has been given access to your vault.
```

Comment:
The client ID isn't required in all managed identity cases (depends on platform and configuration) and `DefaultAzureCredential` won't always use managed identity (for example, when a dev runs this sample on a local machine). I suggest directing people to `DefaultAzureCredential` docs instead of getting into authentication details here.

Reply:
Thanks for catching that, I forgot the environment variable is not necessary in Azure-managed resources (VMs, Azure Functions, etc.)

As for mentioning the use of a managed identity instead of simply referring to the `DefaultAzureCredential` docs, I was under the impression the effort @jlichwa is leading on updating auth samples required us to direct customers to the use managed identity. If we can get away with not telling people to use something in particular and sending them to `azure-identity` I would prefer that, but I will let Jack chime in here.